pylib/cassandra-cqlsh-tests.sh

```diff
@@ -105,46 +109,44 @@ fi
 ccm remove test || true # in case an old ccm cluster is left behind
 ccm create test -n 1 --install-dir=${CASSANDRA_DIR}
 ccm updateconf "enable_user_defined_functions: true"
-
-version_from_build=$(ccm node1 versionfrombuild)
-export pre_or_post_cdc=$(python -c """from distutils.version import LooseVersion
-print (\"postcdc\" if LooseVersion(\"${version_from_build}\") >= \"3.8\" else \"precdc\")
-""")
-case "${pre_or_post_cdc}" in
-    postcdc)
-        ccm updateconf "cdc_enabled: true"
-        ;;
-    precdc)
-        :
-        ;;
-    *)
-        echo "${pre_or_post_cdc}" is an invalid value.
-        exit 1
-        ;;
-esac
-
-ccm start --wait-for-binary-proto
-
-cd ${CASSANDRA_DIR}/pylib/cqlshlib/
+ccm updateconf "cdc_enabled: true"
 
 set +e # disable immediate exit from this point
-pytest
+
+ccm start --wait-for-binary-proto --jvm-version=${java_version}
+
+pushd ${CASSANDRA_DIR}/pylib/cqlshlib/ >/dev/null
+
+pytest --junitxml=${BUILD_DIR}/test/output/cqlshlib.xml
 RETURN="$?"
 
+# don't do inline sed for linux+mac compat
```

Comment:
Don't we need these too - https://github.com/apache/cassandra/blob/trunk/pylib/cassandra-cqlsh-tests.sh#L99-L101? 

Do we use `ant generate-unified-test-report`?

Reply:
No, we don't need to. We can publish as many individual results to Jenkins as we want and Jenkins shows that as a single report